src/libs/actions/PaymentMethods.js

```diff
@@ -103,6 +103,34 @@ function getPaymentMethods() {
         });
 }
 
+function openPaymentsPage() {
+    const onyxData = {
+        optimisticData: [
+            {
+                onyxMethod: 'merge',
```

Comment:
NAB: Looks like we've got a constant for onyxMethod: `CONST.ONYX.METHOD.MERGE`

Reply:
I have a follow up PR for antoher part of this refactor, I'll fix it then